data.access/src/test/java/data/access/DatabaseUtility.java

```diff
@@ -0,0 +1,115 @@
+package data.access;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Types;
+import java.util.ArrayList;
+import java.util.List;
+
+
+import java.sql.ResultSet;
+
+public class DatabaseUtility implements DatabaseAccessor{
+	private String url = "jdbc:mysql://localhost:3306/sakila";
+
+	public List<String> ExecuteSingleColumn(String sql) {
+		List<String> result = new ArrayList<>();
+		try (Connection connection = DriverManager.getConnection(url);
```

Comment:
Each method is connecting to the db. Each method is also duplicating same code in other methods. Maybe find a way to consolidate.